src/sempy_labs/admin/_basic_functions.py

```
@@ -624,12 +927,12 @@ def list_item_access_details(
 
     Parameters
     ----------
-    item_name : str
-        Name of the Fabric item.
-    type : str
-        Type of Fabric item.
+    item : str
+        Name or id of the Fabric item.
+    type : str, default=None
```

type is necessary. need to raise an error if this is not provided.

---

added in the parameters validation and it raises the error.